temporian/core/operators/arithmetic.py

```diff
@@ -0,0 +1,186 @@
+# Copyright 2021 Google LLC.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Arithmetic operator."""
+
+from enum import Enum
+
+from temporian.core import operator_lib
+from temporian.core.data.event import Event
+from temporian.core.data.feature import Feature
+from temporian.core.operators.base import Operator
+from temporian.proto import core_pb2 as pb
+
+
+class ArithmeticOperation(str, Enum):
+    ADDITION = "ADDITION"
+    SUBTRACTION = "SUBTRACTION"
+    MULTIPLICATION = "MULTIPLICATION"
+    DIVISION = "DIVISION"
+
+    @staticmethod
+    def prefix(operation: "ArithmeticOperation") -> str:
+        if operation == ArithmeticOperation.ADDITION:
+            return "sum"
+        if operation == ArithmeticOperation.SUBTRACTION:
+            return "diff"
```

Comment:
nit: make it `"sub"`?

Reply:
Sub could be more misleading than diff, it could mean more things. However, the short version of subtraction (sub) is also a good option. What do you all think? @ianspektor @achoum